CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added legacy homepage image loading test script (DR-2921)
 
+### Updated
+
+- Run homepage images tests 10x in parallel (DR-2937)
```

Comment:
If this is meant to be temporary, I don't think it needs to be included in the changelog.